impl-servlet/src/main/java/org/ocpsoft/rewrite/servlet/RewriteFilter.java

```diff
@@ -144,15 +144,23 @@ public void init(final FilterConfig filterConfig) throws ServletException
 
       if ((configurations == null) || configurations.isEmpty())
       {
+          log.info("configurations are null or empty");
          if (log.isWarnEnabled())
             log.warn("No ConfigurationProviders were registered: " +
                      "Rewrite will not be enabled on this application. " +
                      "Did you forget to create a '/META-INF/services/" + ConfigurationProvider.class.getName() +
                      " file containing the fully qualified name of your provider implementation?");
+      } else {
+          log.info("configuration is ok");
       }
+//       for (ConfigurationProvider<?> configuration : configurations) {
+//           log.info("configuration: " + configuration.toString());
+//           
+//       }
 
       if (log.isInfoEnabled())
-         log.info(Version.getFullName() + " initialized.");
+//         log.info(Version.getFullName() + " initialized.");
+         log.info(Version.getFullName() + " initialized (has support for Jakarta EE9+)003.");
```

Comment:
@rtreacy I think a review from @lincolnthree is coming but in the meantime, he asked over at https://github.com/ocpsoft/rewrite/issues/304#issuecomment-1144954252 the following:

![171656652-91ead015-bca3-43ff-959d-ec99c3ade9a8](https://user-images.githubusercontent.com/21006/171696397-e2abbeff-f8d0-45e3-88ff-cb304b8c8667.png)

Reply:
I figured out how to use the GitHub interface. Review has been started. Thanks!